core/src/main/java/org/apache/gravitino/storage/relational/service/UserMetaService.java

```diff
@@ -285,12 +285,12 @@ public int deleteUserMetasByLegacyTimeline(long legacyTimeline, int limit) {
     SessionUtils.doMultipleWithCommit(
         () ->
             userDeletedCount[0] =
-                SessionUtils.doWithoutCommitAndFetchResult(
+                SessionUtils.getWithoutCommit(
```

Comment:
Why did you change here? `getWithoutCommit` will not return the modified count.

Reply:
> getWithoutCommit will not return the modified count.

why did you say that?